Source/santasyncservice/SNTSyncStage.mm

```diff
@@ -211,8 +218,19 @@ - (NSData *)dataFromRequest:(NSURLRequest *)request timeout:(NSTimeInterval)time
 - (NSError *)performRequest:(NSURLRequest *)request
                 intoMessage:(google::protobuf::Message *)message
                     timeout:(NSTimeInterval)timeout {
-  NSData *data = [self dataFromRequest:request timeout:timeout];
-  if (data.length == 0) return nil;
+  NSError *error;
+  NSData *data = [self dataFromRequest:request timeout:timeout error:&error];
+  if (error) {
+    SLOGE(@"Error performing request: %@", error.localizedDescription);
+    return error;
+  }
+  if (data.length == 0) {
+    NSString *errStr = @"Response is empty";
+    SLOGE(@"%@", errStr);
+    return [NSError errorWithDomain:@"com.google.santa.syncservice"
```

Comment:
Need to fix the test to account for this: SNTSyncTest.mm line 263

Reply:
Will fix in an immediate followup PR